docs/arrebol-worker.md

```diff
@@ -0,0 +1,45 @@
+# Setting up Arrebol worker
+
+## Docker
+
+To configure Docker, follow the steps outlined [here](https://github.com/wesleymonte/worker-deployment). When they are successfully completed, proceed to the instructions below).
+
+## NFS Client
+
+### Dependencies
```

Comment:
Add a simple explanation that these instructions must be executed on the VM (Worker) to install and configure the NFS Client.

Reply:
Done